README.md

```diff
@@ -92,8 +93,8 @@ Diagnostic info for each metric is logged as a console group prepended by the ex
 The kinds of diagnostic info varies per metric. For example, the LCP info includes:
 
 * A reference to the LCP element
-* A table of [LCP sub-part metrics](https://web.dev/optimize-lcp/#:~:text=each%20of%20these-,LCP%20sub%2Dparts,-in%20more%20detail)
-* An optional warning if the tab was [loaded in the background](https://web.dev/lcp/#:~:text=your%20analytics%20service.-,Caution,-Since%20users%20can)
+* A table of [LCP sub-part metrics](https://web.dev/articles/optimize-lcp/#:~:text=each%20of%20these-,LCP%20sub%2Dparts,-in%20more%20detail)
+* An optional warning if the tab was [loaded in the background](https://web.dev/articles/lcp/#:~:text=your%20analytics%20service.-,Caution,-Since%20users%20can)
```

Comment:
```suggestion
* A table of [LCP sub-part metrics](https://web.dev/articles/optimize-lcp#:~:text=each%20of%20these-,LCP%20sub%2Dparts,-in%20more%20detail)
* An optional warning if the tab was [loaded in the background](https://web.dev/articles/lcp#:~:text=your%20analytics%20service.-,Caution,-Since%20users%20can)
```